add mission CRF_TVT115_Lamentin_Limbo to the repository

FluffSquirrel opened the pull request:
sorry for weird formatting copy and pasted it back off a google doc
Mission Pull Request Template

This Pull Request will:

Add the mission Lamentin Limbo made by me to the repository.

Mission Creation Checklist:
[ x] Mission File
[ x] Ingame Play Area Markers
[ x] Ingame Briefings to all players
[x ] Side-based briefings
[x ] Objective Entities
[ x] Mission .conf file

Faction(s):
BLUFOR - Defending
[x ] US
 RHS_US_USMC
 CDF

OPFOR - Attacking
[ x] USSR
 RHS_RF_MSV

INDFOR - /NA
 FIA

Objectives/Win Conditions:
blufor:
defend lamentin harbor - mandatory
defend fob raleigh - optional
opfors the same but just capturing them, the fob is optional for them as well

Terrain: Everon/Eden
Everon/Eden

Short mission description:
outnumbered US forces defend lamentin harbor against USSR forces until an evac ship can arrive (there is obviously not an evac ship in game)

Slotting Ratio:
<3:2> opfor:blufor

Time limit:
<90min>

Mission Briefing Screenshot:
![image](https://github.com/user-attachments/assets/7d514f84-ea59-4804-af11-5c573aeabe93)

Other Notes:

known issues:
~~-cannot figure out how to add a mission timer for the life of me. probably obvious but i looked through everything* and didnt find it
-3:2 ratio isnt mentioned in briefing, i will add it in whenever feedback is given~~

requesting feedback on:
everything obviously but more specifically
-is AO too big? i justify it in that i know 4 (realistically 3) us squads will not be able to cover all of that ground, but by giving them a helo it lets them use their smalled numbers more effectively.
-does the fob have too many props? reforger doesnt have big issues with lots of props, and not at the level the fob has, but if you want the details gone i will remove them. they shouldnt cause problems though. was just having fun building.
-should US heli be unarmed? i intend it to be recon and i mention in US briefing that it should probably be used for recon, but giving it guns felt right, dont know why.
-is 90m timer too long? its got a big AO and everything but i feel like the action will move quickly. guess it just depends on leadership on either side
-does AO border need to be mined. i would think not, but CFC16 2nd mission just showed that what the mission maker sees and what the PLs and COs see arent always lined up lol. i think it will be fine without, ive looked at a lot of angles and cant feasibly see any reason a person could go out of bounds unless it was intentional.
-does opfor need logistics? they have 15 tickets, but to me for a number of tickets that small logi would just be redundant
-i want blufor to be able to build defenses at lamentin, logi squad still has a role in supplying mortars, but as far as building goes its been bugged for a while. should i just scrap it and remove the logi truck, maybe throw in some preplaced fortifications, or will we get a fix/is it fixed?

<Assets? Admin notes? Anything else?>
assets list is in game
mortars need to be spawned at the fob and not at lamentin harbor
lamentin harbor needs an arsenal box with mortar rounds

RandAlThor1 commented:
**QA Feedback**

- BTR crews go at the bottom of their platoon, the callsigns are already setup correctly, just groups are out of order.
![image](https://github.com/user-attachments/assets/d3346402-c5a8-4950-99a2-c0d438dc8231)
- its COY not "1stCOM", and its 1PLT/2PLT  not '1stPLT/2ndPLT"
- vehicles can and should be pre-placed 
- we don't really want recommended plans of action in the briefings 
- give a forward observer to BLUFOR PLT, or could assign it to the pilots to be a FO-A
- technically its still should be TH1 as its a huey

-----------------------

- AO is not too big,
- FOB looks fine, don't think there should be any problems
- I THINK the huey should be fine, BTRs and the like can be pretty effective if they get over eager.
- 90 minutes is probably too long, 60-70 minutes should be fine, important to light a fire under the attackers. (figuratively)
- the mission you referring to showed me I should have expanded the AO around the objectives. I stand by mining is only needed when you want the border to be close to objectives and to be very strict, you have plenty of open space that I think it should be fine.
- I don't really think this needs to be a respawn mission, its pretty straight forward and don't think anyone really benefits from it here.
- building mechanics afaik are pretty broken. scrap logi and just give the mortar team a set amount of shots available to them in the FOB. everyone has a shovel to make trenches if needed.

FluffSquirrel commented:
changelog:
-company/platoon names fixed
-opfor order fixed
-all vehicles and mortars now preplaced
-reccomended plans cut from OPFOR/BLUFOR briefings
-forward observer added to BLUFOR pilot crew
-Helo callsign changed to TH1

-mission time cut to 70m
-all respawns removed, mention of it in sit. brief removed
-both logi vics cut, 
-added prefab ammo at FOB raleigh, 
-both logi and const. trucks removed from assets list, 
-logi team removed, updated callsigns list to boot
-added 1x m923A1 transport & added to assets list
-added 1x m2 humvee and 2x m998 for 4th squad because it was a late add and i forgot to update vics when i added them

RandAlThor1 commented:
**QA Feedback** 
Mission itself looks good.

- Rename the mission config file from "CRF_BT115_Lamentin_Limbo" to CRF_SWP113_Lamentin_Limbo"
- inside the mission config file change the name to the same as above, without the underscores.
- change the description to a quick description of the mission for the loading screen.

otherwise looks good
